### PR TITLE
en,zh: add v1.4.1 release notes

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

- add v1.4.1 release notes

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Review by KanShiori — APPROVED** [review]

File: en/releases/release-1.4.1.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Release date: January 13, 2023
+
+TiDB Operator version: 1.4.1
+
+## New features
+
+- Support cleaning up failed instances of PD, TiKV and TiFlash components by removing the failed Pods and PVCs to handle unplanned failures of Kubernetes nodes in auto failure recovery feature ([#4824](https://github.com/pingcap/tidb-operator/pull/4824), [@lalitkfk](https://github.com/lalitkfk))
+
+    - Need to configure `controllerManager.detectNodeFailure` in TiDB Operator Helm chart and configure `app.kubernetes.io/auto-failure-recovery: "true"` annotation in TidbCluster CR to enable this
```
Comment (ran-huang) [inline en/releases/release-1.4.1.md:15]:
```suggestion
    - To enable this feature, you need to configure `controllerManager.detectNodeFailure` in TiDB Operator Helm chart and configure the `app.kubernetes.io/auto-failure-recovery: "true"` annotation in the TidbCluster CR.
```

File: zh/releases/release-1.4.1.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+发布日期: 2023 年 1 月 13 日
+
+TiDB Operator 版本：1.4.1
+
+## 新功能
+
+- 故障自动转移功能支持在 Kubernetes 节点异常时通过强制移除 Pod 和 PVC 来清理异常的 PD、TiKV 和 TiFlash 节点 ([#4824](https://github.com/pingcap/tidb-operator/pull/4824), [@lalitkfk](https://github.com/lalitkfk))
+
+    - 需要在 TiDB Operator 的 Helm Chart 中配置 `controllerManager.detectNodeFailure` 并在 TidbCluster CR 中配置 `app.kubernetes.io/auto-failure-recovery: "true"` annotation 进行开启
```
[the remark this answers is not in the document]
Comment (csuzhangxc) [reply zh/releases/release-1.4.1.md:15]: 我先给自己记个 issue 吧 #2184

File: zh/releases/release-1.4.1.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## 优化提升
+
+- 支持在 TiDB Operator 的 Helm Chart 中配置 `controllerManager.kubeClientQPS` 与 `controllerManager.kubeClientBurst` 来设置 TiDB Controller Manager 中 Kubernetes client 的 QPS 和 Burst ([#4830](https://github.com/pingcap/tidb-operator/pull/4830)，[@Thearas](https://github.com/Thearas))
```
Comment (csuzhangxc) [inline zh/releases/release-1.4.1.md:19]:
```suggestion
- 支持在 TiDB Operator 的 Helm Chart 中配置 `controllerManager.kubeClientQPS` 与 `controllerManager.kubeClientBurst` 来设置 TiDB Controller Manager 中 Kubernetes client 的 QPS 和 Burst ([#4830](https://github.com/pingcap/tidb-operator/pull/4830), [@Thearas](https://github.com/Thearas))
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**